subsys/net/lib/sockets/mbedtls_dbg.c

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
As there is nothing network specific any more in `mbedtls_dbg.[c|h]`, we could place the files in some more generic place. One such place could be `lib/os`, what do you think?

Reply:
I was about to suggest the same, it seems that module-specific code could also be places in `modules/mbedtls/`.

Reply:
> module-specific code could also be places in `modules/mbedtls/`

+1 to that